Wikipedia/Code/NotificationsCenterViewController.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+
+@objc
+final class NotificationsCenterViewController: ViewController {
+
+	// MARK: - Properties
+
+	var notificationsView: NotificationsCenterView {
+		return view as! NotificationsCenterView
+	}
+
+	let viewModel: NotificationsCenterViewModel
+
+	// MARK: - Lifecycle
+
+	@objc
+	init(theme: Theme, viewModel: NotificationsCenterViewModel) {
+		self.viewModel = viewModel
+		super.init(theme: theme)
+	}
+
+	required init?(coder aDecoder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override func loadView() {
+		super.loadView()
```

Comment:
Great catch! I think their recommendation is slightly misleading – since the default implementation of `loadView` seems to populate `view` with a valid default value, there may be cases where it's not unreasonable to call it (like maybe if your custom view is quite simple and you just want to build it in place).

I think it works as implemented now (though isn't optimized) because it's effectively just assigning a value to `view` twice (in fact, we do it in a [couple](https://github.com/wikimedia/wikipedia-ios/blob/32ca59d18352c20419605c6b469cc530f304948d/Wikipedia/Code/ExploreCardViewController.swift#L57) [other](https://github.com/wikimedia/wikipedia-ios/blob/32ca59d18352c20419605c6b469cc530f304948d/Wikipedia/Code/ReferenceBackLinksViewController.swift#L62) places). But, who knows what side effects lurk in the `UIKit` source – I pushed a commit removing it!